spring-kafka/src/main/java/org/springframework/kafka/listener/KafkaMessageListenerContainer.java

```diff
@@ -83,7 +86,20 @@
 	 * @param topicPartitions the topics/partitions; duplicates are eliminated.
 	 */
 	public KafkaMessageListenerContainer(ConsumerFactory<K, V> consumerFactory, TopicPartition... topicPartitions) {
-		this(consumerFactory, null, null, topicPartitions);
+		this(consumerFactory, null, null, null, topicPartitions);
+	}
+
+	/**
+	 * Construct an instance with the supplied configuration properties, {@link ConsumerRebalanceListener} implementation
+	 * and specific topics/partitions - when using this constructor, {@link #setRecentOffset(long)
+	 * recentOffset} can be specified.
+	 * @param consumerFactory the consumer factory.
+	 * @param consumerRebalanceListener {@link ConsumerRebalanceListener} implementation.
+	 * @param topicPartitions the topics/partitions; duplicates are eliminated.
+	 */
+	public KafkaMessageListenerContainer(ConsumerFactory<K, V> consumerFactory, ConsumerRebalanceListener consumerRebalanceListener,
```

Comment:
As of now corrected this line and going forward will take care. Thanks for pointing me.